tlcc/Makefile

```diff
@@ -6,15 +6,8 @@ TOP = ..
 include $(TOP)/build.mk
 
 LD_LIBRARY_PATH := LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:$$(pwd)/enclave/lib
-SRC := tlcc.go
-OUTPUT := tlcc.so
 
-$(OUTPUT): $(SRC)
-	$(LD_LIBRARY_PATH) $(GO) build -o $@ -buildmode=plugin $<
-
-build: $(OUTPUT)
-
-test: build
+test:
```

Comment:
test has no dependency?  That said, i guess i guess go test handles them all?

Reply:
yep, go test will build everything needed to run `tlcc_test.go`

Reply:
I mean, in theory the dependency is all the go files, but good luck with making a makefile target for that mess. 

Best to let the go toolchain handle it!